process/TI-Gives+Gets.md

```diff
@@ -0,0 +1,52 @@
+| TI Follows security best practices (as recommended by the OpenSSF and others), including passing the OpenSSF Best Practices criteria | May post project updates and tutorials to the OpenSSF blog.  |
+| Project should be integrating with Scorecards  |  |
+|  Begins to establish the appropriate governance that enables its sustainment for potential graduation.| |
+| Projects should be Securing Code Repository -> Managing Contributions Commit Signing , Secret Scanning, Code Scanning (OSFUZZ at a minimum) + Self-assessment Should OpenSSF require these if the SCM supports it, especially using Sigstore? |  |
+
+## Graduated level Gives & Gets
+|     Gives/Requirements             |                Gets/Benefits          |
+| :-----------------------------:    | :-----------------------------------: | 
+|  All requirements of Incubating must be fulfilled and additionally:  | All Gets from Incubating are valid and additionally:  |
+|  Projects must be able to show a consistent release cadence.  |  Receives consideration as in-scope for any submission to an OpenSSF-managed conference or event.  |
+|  Maintains a point of contact for vulnerability reports and follow coordinated vulnerability disclosure practices.  |  Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee).  |
+|  Implements, practices, and refines mature software development and release practices, such as adherence to semantic versioning, and having a declared policy for stable releases and backported fixes.  |  May post project updates and tutorials to the OpenSSF blog.  |
+|  Projects must have documented project governance and be able to demonstrate that governance in action.  |  May request OpenSSF budget for project improvements such as security audits or time-bound contracting needs.  |
+|  When applicable, projects must have completed a security audit through a third party and addressed audit findings and recommendations.  |  May request OpenSSF budget for sustained maintainer stipends (details determined by OpenSSF and project leads).  |
+|   When applicable, Projects should achieve **BLAH** level of SLSA  | With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF.|
+|  |  Projects may use the OpenSSF logo to promote their project (in accordance with the trademark guidelines). Projects may be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName."  |
+|  |  May request considered for Grants  |  
+|  |  May request consideration to get Contract Developers  |
+|  |  Requests for one time funding needs to include: Tech writer, Graphic designer, Security audit, Event support, Outreach, Dashboard/reports, Recognition awards, Infrastructure support for software projects (like special build, QA) (will have ongoing operational expense),  Cloud hosting (will have ongoing operational expense)  |
```

Comment:
A lot of this feels like it should be under incubating.  If the goal of graduation from OpenSSF end is to have a well structured, well governed, well written, well documented project some of these projects are coming to OpenSSF to support there. I think things like security audits and documentation in particular should be things that OpenSSF helps provide to take a project from incubating to graduation.

Reply:
sounds fine to me, do you want to suggest how/where to move the text?